.github/workflows/updates-e2e.yml

```
@@ -0,0 +1,82 @@
+name: Updates e2e
+
+on:
+  workflow_dispatch: {}
+  pull_request:
+    paths:
+      - .github/workflows/updates-e2e.yml
+      - packages/expo-json-utils/**
+      - packages/expo-manifests/**
+      - packages/expo-modules-core/**
+      - packages/expo-structured-headers/**
+      - packages/expo-updates-interface/**
+      - packages/expo-updates/**
+      - packages/expo/**
+  push:
+    branches: [main, 'sdk-*']
+    paths:
+      - .github/workflows/updates-e2e.yml
+      - packages/expo-json-utils/**
+      - packages/expo-manifests/**
+      - packages/expo-modules-core/**
+      - packages/expo-structured-headers/**
+      - packages/expo-updates-interface/**
+      - packages/expo-updates/**
+      - packages/expo/**
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event_name }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  android:
+    runs-on: macos-11
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+        with:
+          submodules: false
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - uses: actions/cache@v2
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+      - run: yarn install --frozen-lockfile
+      - uses: actions/cache@v2
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('android/*.gradle*') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+      - run: yarn global add expo-cli
+      - run: echo "$(yarn global bin)" >> $GITHUB_PATH
+      - name: Init new expo app
+        working-directory: ../
+        run: expo-cli init updates-e2e --yes
+      - name: Add local expo-updates and dependencies
+        working-directory: ../updates-e2e
+        run: yarn add file:../expo/packages/expo-updates file:../expo/packages/expo file:../expo/packages/expo-modules-core file:../expo/packages/expo-json-utils file:../expo/packages/expo-manifests file:../expo/packages/expo-structured-headers file:../expo/packages/expo-updates-interface
+      - name: Setup app.config.json
+        working-directory: ../updates-e2e
+        run: echo "{\"name\":\"updates-e2e\",\"plugins\":[\"expo-updates\"],\"android\":{\"package\":\"dev.expo.updatese2e\"},\"ios\":{\"bundleIdentifier\":\"dev.expo.updatese2e\"}}" > app.config.json
+      - name: Prebuild
+        working-directory: ../updates-e2e
+        run: expo-cli prebuild
+      # TODO: remove once the local template projects are using SDK 45 packages
+      - name: Manually bump kotlin version (workaround)
+        working-directory: ../updates-e2e
+        run: sed -i -e 's/\(buildToolsVersion\ =\ \"[0-9.]*\"\)/\1\nkotlinVersion\ =\ \"1\.6\.10\"/' ./android/build.gradle
```

Most of these backslashes look unnecessary, especially in the replacement pattern:
```suggestion
        run: sed -i -e 's/\(buildToolsVersion = "[0-9.]*"\)/\1\nkotlinVersion = "1.6.10"/' ./android/build.gradle
```